src/13_file_io.py

```diff
@@ -9,10 +9,17 @@
 # Print all the contents of the file, then close the file
 
 # YOUR CODE HERE
-
+filename = "./foo.txt"
+file = open(filename, "r")
+for line in file:
+   print(line)
+file.close()
```

Comment:
Good job here. Another way to access a file is with `with open("file name here") as file: ...`. Cool thing is that we don't need to write `file.close()` with `with open()` as it is called automatically.